assets/javascripts/discourse/components/gamification-leaderboard.js

```diff
@@ -25,6 +25,12 @@ export default Component.extend({
 
   @discourseComputed("model.users.[]")
   ranking(users) {
+    //im sorry for this… it's all I could think of :(
```

Comment:
This is fine. I would change it so that you slice first and then run the `forEach` loop. That way you're only looping an array with 3 elements.

Reply:
It's returning everything except the first 3. Atleast I hope it does. .